packit/cli/packit_base.py

```diff
@@ -55,11 +55,17 @@ def get_command(self, ctx, cmd_name):
         "If this is not specified, default to the first remote."
     ),
 )
+@click.option(
+    "-c",
+    "--config",
+    "package_config_path",
+    help="Path to package configuration file.",
```

Comment:
I wonder if this gonna be understood by users well: I would probably at least mention that it defaults to `.packit.yaml` and `packit.yaml`.